### ISSUE TITLE
torchvision.utils.make_grid modifies input tensor in place

### ISSUE THREAD

**Issue by cwkx** [pytorch/vision#217]
Was using `torchvision.utils.make_grid `to view model weights, but turns out this function modifies the input tensor in-place, causing weirdness during training. 

Would be great if this behaviour was commented (if this is the desired behaviour for normalising it), or changed to only modify output grid & not use in-place normalisation.

**Comment by fmassa** [comment]
Indeed, this doesn't look like a good default behaviour. Could you send a PR? Maybe just adding a `.clone()` in the beginning of the function?

**Comment by fmassa** [comment]
I sent a PR addressing this issue in #241.
